test/passing/types.ml

```diff
@@ -50,3 +50,10 @@ val x :
     (* boooooom *)
     (** foooooooooooooooo foooooooooooooooooooooooo fooooooooooooooooooooooo
         fooooooooooooooooooo fooooooooooooooo*) ]
+
+type voting_period =
+  Tezos_client_alpha.Proto_alpha.Alpha_context.Voting_period.kind =
```

Comment:
I'm unsure if this is the best choice of indentation level. Aligning the type alias with the definition, where the `=` is all the way on the right, is possible misleading. What about increasing the indentation of the type alias, perhaps aligning with the beginning of the defined type name?

It would also be good to check what happens to docstrings and comments in these cases.

Reply:
Fixed in my last commit.